dipy/viz/streamline.py

```diff
@@ -31,6 +36,14 @@ def show_bundles(bundles, interactive=True, view='sagital', colors=None,
         Default is None.
 
     """
+
+    if view == "sagital":
```

Comment:
can you add `# codespell:ignore` even if it is not taken into account yet

Reply:
Added in b5be3fc.